perun-base/src/test/resources/test-schema.sql

```diff
@@ -1314,6 +1314,24 @@ create table blacklists (
 	constraint bllist_user_fk foreign key (user_id) references users(id)
 );
 
+create table vos_bans (
+	id integer not null,
+	member_id integer not null,
+	vo_id integer not null,
+	description longvarchar,
+	banned_to timestamp not null,
+	created_at timestamp default current_date not null,
+	created_by longvarchar default user not null,
+	modified_at timestamp default current_date not null,
+	modified_by longvarchar default user not null,
+	created_by_uid integer,
+	modified_by_uid integer,
+	constraint vos_bans_pk primary key (id),
+	constraint vos_bans_u unique (member_id),
+	constraint vos_bans_mem_fk foreign key (member_id) references members (id),
+	constraint vos_bans_vo_fk foreign key (vo_id) references vos (id)
+);
+
```

Comment:
You should also remove "suspendedTo" from members table.

Reply:
done